src/agora/radio_lib.cpp

```diff
@@ -247,34 +247,48 @@ void RadioConfig::configureBSRadio(RadioConfigContext* context)
         baStn[i]->setFrequency(SOAPY_SDR_TX, ch, "RF", _cfg->radioRfFreq);
         baStn[i]->setFrequency(SOAPY_SDR_TX, ch, "BB", kUseUHD ? 0 : _cfg->nco);
 
-        if (!kUseUHD) {
-            if (info["frontend"].find("CBRS") != std::string::npos) {
-                if (_cfg->freq > 3e9)
-                    baStn[i]->setGain(SOAPY_SDR_RX, ch, "ATTN", -6); //[-18,0]
-                else if (_cfg->freq > 2e9 && _cfg->freq < 3e9)
-                    baStn[i]->setGain(SOAPY_SDR_RX, ch, "ATTN", -18); //[-18,0]
-                else
-                    baStn[i]->setGain(SOAPY_SDR_RX, ch, "ATTN", -12); //[-18,0]
-                baStn[i]->setGain(SOAPY_SDR_RX, ch, "LNA2", 17); //[0,17]
-            }
+        // Unified gains for both lime and frontend
+        if (_cfg->single_gain()) {
```

Comment:
single gain is only valid for Iris case. The 'if (!kUseUHD)` should be outside of this `if`.

Reply:
@tingjunchen Can you please review my changes? Should I change as Rahman suggested?

Reply:
@rdoosty I have changed according to your comments.